### DISCUSSION TITLE
[v3][ wire:model="dataItem.slug" ] not get value on property model laravel

### DISCUSSION THREAD

**Discussion by devhau** [livewire/livewire#6039] [7. Bugs (archived)]
<img width="793" alt="image" src="https://github.com/livewire/livewire/assets/50167033/b7ad3417-0e4e-4a2d-90bb-ca33188b22e2">
<img width="793" alt="image" src="https://github.com/livewire/livewire/assets/50167033/b0dab371-2cc5-488a-a7ef-0272ca4a64c0">
<img width="793" alt="image" src="https://github.com/livewire/livewire/assets/50167033/a1c4f2ee-8d8a-42eb-bbe3-3d3f7980940d">

**Comment by joshhanley** [comment]
@devhau V3 doesn't support model binding by default. As such we recommend using `FormObjects` instead (see [FormObjects](https://livewire.laravel.com/docs/forms#extracting-a-form-object)) but you can enable legacy model binding support (V2 implementation) if you're trying to upgrade an existing app, by changing the `livewire.legacy_model_binding` config to `true`, to give you time to upgrade to using `FormObjects`.

Hope this helps!
